db/migrate/20230127102334_add_monthly_charges_value_check.rb

```diff
@@ -0,0 +1,7 @@
+class AddMonthlyChargesValueCheck < ActiveRecord::Migration[7.0]
+  def change
+    change_table :sales_logs, bulk: true do |t|
+      t.column :monthly_charges_value_check, :integer
+    end
+  end
+end
```

Comment:
purely out of interest, is there a benefit in migrations to doing a change_table over just an add_column?
also, is the bulk: true doing anything here? When I investigated it looked to me from docs that all it does is group together the changes within a change_table, so if there's only one then it won't be doing anything as far as I can tell.

Reply:
Yeah, it doesn't really matter when it's just one change, it's just a different syntax at that point